### PR TITLE
[SIW-1560] Adding the email template for WI activation

### PR DESCRIPTION
**Pull request by christianatzeni**
I added the email template that is sent to the user when he creates a new wallet instance.

#### List of Changes

- Created a template (file index.mjml)
- Created the related test (file index.test.ts)
- Created the applier (file applier.template.ts)
- Added the generate:walletInstanceActivation command

#### Motivation and Context

The change is necessary because the user who created and activated a new wallet instance must be able to receive an email confirming it and advising that it can be revoked at any time.

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "generate:loginnotification": "ts-node ./utils/email_generation_script.ts LoginNotificationIOWeb/",
     "generate:loginnotificationfallback": "ts-node ./utils/email_generation_script.ts LoginNotification/",
     "generate:fimsaccessexport": "ts-node ./utils/email_generation_script.ts FimsAccessExport/",
+    "generate:walletInstanceActivation": "ts-node ./utils/email_generation_script.ts WalletInstanceActivation/",
```
Comment (silvicir) [inline package.json:34]: walletinstancecreation
```suggestion
    "generate:walletinstancecreation": "ts-node ./utils/email_generation_script.ts WalletInstanceCreation/",
```

File: WalletInstanceCreation/__test__/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+describe("Snapshot testing", () => {
+  afterEach(() => {
+    vi.restoreAllMocks();
+  });
+
+  it("should match snapshot", () => {
+    const htmlOutput: string = apply(
+      { href: "https://test.test/" } as ValidUrl,
+      { href: "https://test.test/" } as unknown as ValidUrl
```
Comment (silvicir) [inline WalletInstanceCreation/__test__/index.test.ts:13]: why `unknown`?

File: WalletInstanceCreation/__test__/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { describe, it, expect, vi, afterEach } from "vitest";
+import { ValidUrl } from "@pagopa/ts-commons/lib/url";
+import { apply } from "../";
```
Comment (silvicir) [inline WalletInstanceCreation/__test__/index.test.ts:3]: are you sure this is right?

File: WalletInstanceCreation/__test__/index.test.ts (hunk 2 above)
Comment (christianatzeni) [reply WalletInstanceCreation/__test__/index.test.ts:13]: typo

File: WalletInstanceCreation/index.mjml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            align="left"
+            color="#17324D"
+            padding-top="0px"
+            padding-bottom="0px"
+            padding-left="0px"
+            padding-right="0px"
+          >
+            <p>
+              Ciao,<br /><br />
+              il tuo portafoglio sull'app IO è pronto a
```
Comment (silvicir) [inline WalletInstanceCreation/index.mjml:54]:
```suggestion
              il tuo Portafoglio sull'app IO è pronto a
```

**Review by silvicir — APPROVED** [review]

**Pull request merged**